README.yaml

```
@@ -130,31 +59,35 @@ usage: |-
   (which tests and deploys the example on AWS), see [test](test).
 
   ```hcl
-  # Create a standard label resource. See [null-label](https://github.com/cloudposse/terraform-null-label/#terraform-null-label--)
-  module "label" {
-    source  = "cloudposse/label/null"
-    # Cloud Posse recommends pinning every module to a specific version, though usually you want to use the current one
-    # version = "x.x.x"
-
-    namespace = "eg"
-    name      = "example"
-  }
-
-  module "example" {
-    source  = "cloudposse/*****/aws"
+  module "zone_route53" {
```

nit

```suggestion
  module "zone" {
```

or

```suggestion
  module "route53_zone" {
```

would read better. Please apply the same change to the example.